### DISCUSSION TITLE
为什么vs2022编译通过不能运行

### DISCUSSION THREAD

**Discussion by fivegg** [Jeric-X/SyncClipboard#156] [General]
使用release下载的程序就可以直接运行。自己编译的就不行
[image](https://github.com/user-attachments/assets/a0d11b2f-83ef-4dee-9fd0-2a6e18d16022)
![企业微信截图_20241212181227](https://github.com/user-attachments/assets/04de3e9c-bf55-4270-91d9-6eec1839d83c)

按照说明安装.net 6 等，也安装了程序链接的windows app runtime.

**Comment by Jeric-X** [comment]
这就是Windows app runtime没安装对
或者把这行改成true，这样vs编译的时候会把windows app runtime相关的dll一起打包
https://github.com/Jeric-X/SyncClipboard/blob/35084417378bdf81fe99bdb7664eb19667a8d868/src/SyncClipboard.WinUI3/SyncClipboard.WinUI3.csproj#L12
